spec/sign_up_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+def sign_up(user)
+  visit '/'
```

Comment:
```suggestion
  visit root_path
```

Try to use rails pathing rather than hardcoding values